tungsten_tempest_plugin/tests/api/contrail/rbac_base.py

```diff
@@ -110,6 +110,12 @@ class BaseContrailTest(rbac_utils.RbacUtilsMixin, test.BaseTestCase):
 
     required_contrail_version = None
 
+    @classmethod
+    def _get_credentials_provider(cls):
+        prov = super(BaseContrailTest, cls)._get_credentials_provider()
+        prov.extra_roles = ['admin']
```

Comment:
Have you tried to run the tests without adding `admin` role here?
The Patrole always run the tests with `admin` role and overrider the role to a target role (`member`, `reader`) when `override_role` function is called.
https://github.com/openstack/patrole/blob/master/patrole_tempest_plugin/rbac_utils.py#L298